molecule/cluster/playbook.yml

```diff
@@ -2,8 +2,6 @@
 
 - hosts: all
   any_errors_fatal: true
-  roles:
```

Comment:
`roles` was put before `vars` on purpose. This way we ensure better readability since `vars` is meant to be extended and this way we can only append to the end of file.

Reply:
This is just aestetic. We can switch back if you think it's better.